avm/utl/types/avm-common-types/tests/e2e/import/main.test.bicep

```diff
@@ -0,0 +1,278 @@
+targetScope = 'subscription'
+
+metadata name = 'Import all'
+metadata description = '''
+This example imports all available types of the given module.
+
+Note: In your module you would import only the types you need.
+'''
+
+// ============== //
+// Test Execution //
+// ============== //
+
+import {
+  customerManagedKeyType
+  diagnosticSettingFullType
+  diagnosticSettingLogsOnlyType
+  diagnosticSettingMetricsOnlyType
+  roleAssignmentType
+  lockType
+  managedIdentityAllType
+  managedIdentityOnlySysAssignedType
+  managedIdentityOnlyUserAssignedType
+  privateEndpointMultiServiceType
+  privateEndpointSingleServiceType
+  secretToSetType
+  secretSetType
+} from '../../../main.bicep' // Would be: br/public:avm/utl/types/avm-common-types:<version>
+
+//  ====================== //
+//   Diagnostic Settings   //
+//  ====================== //
+param diagnosticFull diagnosticSettingFullType[] = [
```

Comment:
Note @eriqua, I changed the test now to show both a parameter (the common use case) and the output (which shows how to use it in an output yes, but also ensures we don't see 500 warnings that xzy param would not be used 😄).

Also updated note test's name & description.

Please resolve if acknowledged